app/templates/docs/action.html

```diff
@@ -1,615 +1,643 @@
-<li><a class="reference internal" href="#get-1-actions-idaction-card-field">GET /1/actions/[idAction]/card/[field]</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-display">GET /1/actions/[idAction]/display</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-entities">GET /1/actions/[idAction]/entities</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-list">GET /1/actions/[idAction]/list</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-list-field">GET /1/actions/[idAction]/list/[field]</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-member">GET /1/actions/[idAction]/member</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-member-field">GET /1/actions/[idAction]/member/[field]</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-membercreator">GET /1/actions/[idAction]/memberCreator</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-membercreator-field">GET /1/actions/[idAction]/memberCreator/[field]</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-organization">GET /1/actions/[idAction]/organization</a></li>
-<li><a class="reference internal" href="#get-1-actions-idaction-organization-field">GET /1/actions/[idAction]/organization/[field]</a></li>
-<li><a class="reference internal" href="#put-1-actions-idaction">PUT /1/actions/[idAction]</a></li>
-<li><a class="reference internal" href="#put-1-actions-idaction-text">PUT /1/actions/[idAction]/text</a></li>
-<li><a class="reference internal" href="#delete-1-actions-idaction">DELETE /1/actions/[idAction]</a></li>
-</ul>
-</li>
-</ul>
+<span id="index"></span>
+<h1 id="action">action<a class="headerlink" href="#action" title="Permalink to this headline"></a></h1>
+<p>Actions are generated whenever an action occurs in Trello. For instance, when a user deletes a card, a deleteCard action is generated and includes information about the deleted card, the list the card was in, the board the card was on, the user that deleted the card, and the idObject of the action. Actions for Trello objects can be listed from nested action endpoints - e.g. the resource <a href="/advanced-reference/board#get-1-boards-board-id-actions">GET board/[board_id]/actions</a> lists all of the actions for the given board. Additionally, data regarding individual action objects can be retrieved and updated using the resources listed below.</p>
+<p>Included at the bottom of this page are a <a href="#all-action-types">list of all action types</a> and <a href="#excluded-action-types">action types that are not returned</a> from nested resources.</p>
+  <div class="section" id="get-1-actions-idaction">
+    <h2>GET <span class="target" id="actions-idaction">/1/actions/[idAction]</span><a class="headerlink" href="#get-1-actions-idaction" title="Permalink to this headline"></a></h2>
```

Comment:
And maybe another example showing some extra options

`GET /1/actions/560bf78ea40a4590ca3c5c7d?display=true&entities=true`

```json

    "id": "560bf78ea40a4590ca3c5c7d",
    "idMemberCreator": "556c8537a1928ba745504dd8",
    "data": {
        "list": {
            "name": "Washington",
            "id": "560bf48efe2771efe9b45997"
        },
        "board": {
            "shortLink": "tBmYPSYe",
            "name": "US National Parks",
            "id": "560bf4298b3dda300c18d09c"
        },
        "card": {
            "shortLink": "1UtQeVnQ",
            "idShort": 53,
            "id": "560bf587882bb89c3882294e",
            "name": "Mount Rainier National Park"
        },
        "old": {
            "name": "Mount Rainier National Park | MapQuest National Parks"
        }
    },
    "type": "updateCard",
    "date": "2015-09-30T14:54:06.281Z",
    "memberCreator": {
        "id": "556c8537a1928ba745504dd8",
        "avatarHash": "0b48c7057767cca8f339109b27a064d7",
        "fullName": "Matt Cowan",
        "initials": "MC",
        "username": "mattcowan"
    },
    "entities": [
        {
            "type": "member",
            "id": "556c8537a1928ba745504dd8",
            "username": "mattcowan",
            "text": "Matt Cowan"
        },
        {
            "type": "text",
            "text": "renamed"
        },
        {
            "type": "card",
            "id": "560bf587882bb89c3882294e",
            "shortLink": "1UtQeVnQ",
            "text": "Mount Rainier National Park"
        },
        {
            "type": "text",
            "text": "(from Mount Rainier National Park | MapQuest National Parks)"
        }
    ],
    "display": {
        "translationKey": "action_renamed_card",
        "entities": {
            "card": {
                "type": "card",
                "id": "560bf587882bb89c3882294e",
                "shortLink": "1UtQeVnQ",
                "text": "Mount Rainier National Park"
            },
            "name": {
                "type": "text",
                "text": "Mount Rainier National Park | MapQuest National Parks"
            },
            "memberCreator": {
                "type": "member",
                "id": "556c8537a1928ba745504dd8",
                "username": "mattcowan",
                "text": "Matt Cowan"
            }
        }
    }
}
```